src/components/Editor/Editor.js

```diff
@@ -0,0 +1,45 @@
+export default function Editor({ $target, initialState, onEditing }) {
+  const $editor = document.createElement('form');
+  $editor.className = 'editor';
+  $target.appendChild($editor);
+
+  const $title = document.createElement('input');
+  $title.className = 'title';
+  $title.name = 'title';
+  $title.placeholder = '제목을 작성하세요...';
+
+  const $content = document.createElement('textarea');
+  $content.className = 'content';
+  $content.name = 'content';
+  $content.placeholder = '내용을 입력하세요...';
+
+  let isInitialize = false;
+
+  this.state = initialState;
+  this.setState = (nextState) => {
+    this.state = nextState;
+    this.render();
+  };
+
+  $editor.addEventListener('keyup', (e) => {
+    const name = e.target.getAttribute('name');
+    if (this.state[name] !== undefined) {
```

Comment:
p3;
```suggestion
    if (this.state[name]) {
```
[truthy](https://developer.mozilla.org/ko/docs/Glossary/Truthy)로 처리 가능할 것 같네요